service/src/main/java/com/commercetools/pspadapter/payone/mapping/PayoneRequestFactory.java

```diff
@@ -47,19 +61,22 @@ protected void mapFormPaymentWithCartLike(final AuthorizationRequest request,
         try {
             MappingUtil.mapCustomerToRequest(request, ctPayment.getCustomer());
         } catch (final IllegalArgumentException ex) {
-            logger.debug("Could not fully map payment with ID {} {}", paymentWithCartLike.getPayment().getId(), ex.getMessage());
+            logger.debug("Could not fully map payment with ID {} {}",
+                    paymentWithCartLike.getPayment().getId(), ex.getMessage());
```

Comment:
Does ex.getMessage() contains more details on exact error reason as "Could not fully map payment with ID" does not seem very informativ?